src/main/java/sirius/biz/cluster/InterconnectCacheCoherence.java

```diff
@@ -59,6 +71,12 @@ public void handleEvent(JSONObject event) {
         } else if (Strings.areEqual(type, TYPE_REMOVE)) {
             String key = event.getString(MESSAGE_KEY);
             CacheManager.removeCoherentCacheKeyLocally(cache, key);
+        } else if (Strings.areEqual(type, TYPE_PUT)) {
+            String node = event.getString(MESSAGE_NODE);
+            if (!Strings.areEqual(node, CallContext.getNodeName())) {
+                String key = event.getString(MESSAGE_KEY);
+                CacheManager.removeCoherentCacheKeyLocally(cache, key);
```

Comment:
maybe add a line comment to indicate that this is the intentional behaviour as deleting is cheaper and more efficient thant emulating a put via an extra lookup...